### PR TITLE
feature: ZENKO-315 Add NFS property to BucketInfo

### PR DESCRIPTION
**Pull request by bennettbuchanan**

### CODE REVIEW

**Review by philipyoo — DISMISSED** [review]
lgtm, do we need to update `modelVersion` number?

**Review by ploki — DISMISSED** [review]

File: lib/models/ObjectMD.js [hunk 1]
```diff
@@ -706,7 +707,7 @@ class ObjectMD {
      */
     setReplicationInfo(replicationInfo) {
         const { status, backends, content, destination, storageClass, role,
-            storageType, dataStoreVersionId } = replicationInfo;
+            storageType, dataStoreVersionId, isNFS } = replicationInfo;
```
Comment (jonathan-gramain) [inline lib/models/ObjectMD.js:713]: `fromNFS` could read better, along with `setReplicationFromNFS()`?

Comment (ploki) [reply lib/models/ObjectMD.js:713]: we "cross repository" agreed on "isNFS" to carry the fact that a bucket might be exported with NFS.

**Review by jonathan-gramain — DISMISSED** [review]

**Comment by jonathan-gramain** [comment]
@bert-e wait

**Review by jonathan-gramain — DISMISSED** [review]

**Review by jonathan-gramain — APPROVED** [review]

**Review by philipyoo — APPROVED** [review]

**Pull request merged**